src/components/StoryPointsContainer.tsx

```diff
@@ -0,0 +1,79 @@
+'use client'
+
+import { socketRoomEmitter } from '@/services/socket'
+import { useSocketListener } from '@/services/socket'
+import { useState, useEffect } from 'react'
+import StoryPointButton from './StoryPointButton'
+import type { RoomInfoData } from './RoomMainUi'
+
+// export default function StoryPointsContainer({
+// 	setSelectedStoryPoint,
+// }: { setSelectedStoryPoint: (value: string) => void }) {
+export default function StoryPointsContainer({
+	roomId,
+	userName,
+}: RoomInfoData) {
+	const [selectedStoryPoint, setSelectedStoryPoint] = useState<string | null>(
+		null,
+	)
+	const [isPointBtnDisabled, setIsPointBtnDisabled] = useState(false)
+
+	// const resetStoryPointBtns = useSocketListener('reset-points')
+	const resetStoryPointBtns = useSocketListener('show-disable-reset-points')
+
+	useEffect(() => {
```

Comment:
todo/bug: Why do you need these `useEffects`? I believe you don't need any of these two, and it's making your component more complex. Can you find a better way? :) We can pair in our next call.

Reply:
I've reworked this to remove the useEffects. I was getting very confused during our call because you had put the socketEmit in the socketListener, but this page only needs an Emitter when a button is clicked. Not when it receives the reset "boolean". I did a commit for this

Reply:
See below. Changing this back to one of the useEffects. We will have to discuss more later. I'll try to send a video explaining what I'm seeing.